projects/storefrontlib/src/lib/my-account/order/order-details/order-details.component.spec.ts

```diff
@@ -58,7 +59,7 @@ const mockOrder = {
 })
 class MockOrderSummaryComponent {
   @Input()
-  cart: any;
+  cart: Cart;
```

Comment:
Maybe we can remove it completely, if this test doesn't need it

Reply:
Causes errors. See above